cmd/router/main.go

```diff
@@ -56,15 +57,16 @@ func callService(serviceUrl string, input []byte, headers http.Header) ([]byte,
 	resp, err := http.DefaultClient.Do(req)
 
 	if err != nil {
-		log.Error(err, "An error has occurred from service", "service", serviceUrl)
-		return nil, err
+		log.Error(err, "An error has occurred while calling service", "service", serviceUrl)
+		return nil, 500, err
 	}
 	defer resp.Body.Close()
 	body, err := io.ReadAll(resp.Body)
 	if err != nil {
 		log.Error(err, "error while reading the response")
 	}
-	return body, err
+	log.Info("Exiting callService", "serviceUrl", serviceUrl)
```

Comment:
Should this be at debug level ?

Reply:
@yuzisun : I can do that. But 1 quick thing, is it the only to do debug logging with controller-runtime's log package ?
```
log.V(-1).Info("Exiting callService", "serviceUrl", serviceUrl)
```

Reply:
I have entry log at info.
Another thing is , I will fix the logging by moving uber-zap logger that is more flexible in a separate PR.

Reply:
we can update to a more useful logging with the time spent on the service call

Reply:
actually we already have `defer timeTracker` to log the info so we can remove this log line